app/src/main/java/com/example/stardewvalleypockettracker/BaseActivity.java

```diff
@@ -0,0 +1,42 @@
+package com.example.stardewvalleypockettracker;
+
+import android.os.Bundle;
+import android.view.Menu;
+import android.view.MenuInflater;
+import android.widget.ImageView;
+import android.widget.TextView;
+
```

Comment:
Is there a reason for the extra whitespace here?

Reply:
No, but let's clean it up in a future commit